src/components/ProductList.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
+import Carrousel from './Carrousel';
```

Comment:
don't break the Single Responsibility Principle, the functionality of carrousel is not useful within ProductList, remember to keep components smalls and function-specific. Create a page component for the / route (something like pages/HomePage.js) with these two components inside.

https://en.wikipedia.org/wiki/Single_responsibility_principle
https://en.wikipedia.org/wiki/SOLID